spec/models/generic_file_spec.rb

```diff
@@ -744,4 +744,18 @@ def paranoid_edit_permissions
       described_class.where_public
     end
   end
+  describe "to_csv" do
+    before do
+      @file.title = ["My Title"]
+      @file.creator = ["Von, Creator"]
+      @file.resource_type = ["Book", "Other"]
+      @file.rights = ["Mine"]
+      @file.mime_type = "application/pdf"
+    end
+    subject { @file.to_csv }
+    it do
```

Comment:
Maybe a minor reformat would suffice (there are currently 0 `it do`s in the codebase):

```ruby
it 'parses into an array' do
  expect(subject).to eq 'My Title,...'
  expect(CSV.parse(subject)).to eq [['My Title', ... ]]
end
```

And do we want that to return an array of arrays, or just an array?